modules/test_react_instrument/jsx/InputDate.js

```diff
@@ -0,0 +1,39 @@
+import InputBase from "./InputBase";
+
+export default class InputDate extends InputBase {
```

Comment:
Composition is strongly preferred over inheritance in React:
https://facebook.github.io/react/docs/composition-vs-inheritance.html
http://stackoverflow.com/a/36994501/2070413